### PR TITLE
add prac 9 files

### PR DESCRIPTION
**Pull request by Lijiayi2**
@gzy20020626
@nora-zyy
@AdriantoHartono

### CODE REVIEW

**Comment by AdriantoHartono** [comment]
Great code, all the code has been written neatly in each function🙏

File: prac_09/taxi_test.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    # Restart the meter and then drive the car 100 km
+    my_taxi.start_fare()
+    my_taxi.drive(100)
+
+    # Print the details and the current fare
+    print(my_taxi)
+    print("Current fare: ${:.2f}".format(my_taxi.get_fare()))
+
+if __name__ == '__main__':
+    main()
```
Comment (nora-zyy) [inline prac_09/taxi_test.py:23]: The code writing approach is very clear and coherent.

File: prac_09/unreliable_car_test.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+        print(f"\nDrive {i} km:")
+        print(f"{good_car.name} drove {good_car.drive(i)} km")
+        print(f"{bad_car.name} drove {bad_car.drive(i)} km")
+
+    # print the final states of the cars
+    print(f"\n{good_car.name} final state:\n{good_car}")
+    print(f"\n{bad_car.name} final state:\n{bad_car}")
+
+
+main()
```
Comment (nora-zyy) [inline prac_09/unreliable_car_test.py:20]: The test code is written clearly and in detail.

Comment (AdriantoHartono) [reply prac_09/unreliable_car_test.py:20]: Nice code

File: prac_09/taxi_test.py (hunk 1 above)
Comment (AdriantoHartono) [reply prac_09/taxi_test.py:23]: Great👍

**Pull request merged**